### PR TITLE
US35 - Admin Invoices

### LINKED ISSUE #40: US 35: Admin Invoices

### PR DESCRIPTION
**Pull request by AustinKCodes**
US35 with tests
Shows the total revenue for the current invoice

### CODE REVIEW

File: app/controllers/admin/invoices_controller.rb [hunk 1]
```diff
@@ -6,5 +6,6 @@ def index
   def show
     @invoice = Invoice.find(params[:id])
     @invoice_items = @invoice.invoice_items.includes(:item)
+    @total_revenue = @invoice.invoice_items.sum { |item| item.quantity * item.unit_price}
```
Comment (zach-bergman) [inline app/controllers/admin/invoices_controller.rb]: I think this should be a method in the Invoice model using Active Record!

**Review by zach-bergman — CHANGES_REQUESTED** [review]
Could you make the total_revenue method in the invoice model using AR? I think we're supposed to keep this sort of logic/calculations in the model, then just call the model method inside the controller

File: app/controllers/admin/invoices_controller.rb (hunk 1 above)
Comment (AuroraTuring) [reply app/controllers/admin/invoices_controller.rb:9]: Good idea!

**Review by AuroraTuring — CHANGES_REQUESTED** [review]
Zach's idea should be implemented.

**Review by noahdurbin — COMMENTED** [review]
looks great! I do agree with Zach and think we could move that method into the model

**Review by AuroraTuring — APPROVED** [review]

**Review by zach-bergman — APPROVED** [review]

**Review by noahdurbin — APPROVED** [review]

**Pull request merged**